src/layouts/sidebar-sidecar/server.ts

```diff
@@ -36,30 +34,6 @@ const BASE_PATHS_TO_NAMES = {
   plugins: 'Plugins',
 }
 
-// This Remark plugin rewrites img URLs from our Marketing Content Server API
-// to Dev Portal's next/image optimization endpoint.
-function remarkRewriteImageUrls() {
-  return function plugin() {
-    return function transformTree(tree) {
-      visit<Image, string>(tree, 'image', (node) => {
-        if (node.url.includes(process.env.MKTG_CONTENT_API)) {
-          const params = new URLSearchParams()
-          params.set('url', node.url)
-          // next/image requires that we specify an allowed width. The Dev
-          // Portal docs page renders images at 896 pixels wide. To support high
-          // DPI displays, we double this value to 1792, and round up to the
-          // nearest supported width of 1920.
-          params.set('w', '1920')
-          // By default, next/image uses a quality setting of 75.
-          params.set('q', '75')
-
-          node.url = `/_next/image?${params.toString()}`
-        }
-      })
-    }
-  }
-}
```

Comment:
Do you think we should still do this for images without dimensions specified? Or just consider the next/image usage as a progressive enhancement, especially as more content continues to get uploaded with dimensions in the image URLs.

Reply:
My thinking here was that we could manually trigger a content sync for other versions, so it wouldn't be worth the effort to maintain this remark plugin alongside the new component-based approach.

That being said, if there's some reason we can't easily sync old versions, I think I'd rather do this URL rewrite in the `Image` component as opposed to a remark plugin given how much easier it would be for others to discover.